tests/test-sources/plugins/languages/treesitter/treesitter.nix

```diff
@@ -1,14 +1,55 @@
+{ pkgs, ... }:
 {
+  default = {
+    plugins.treesitter = {
+      enable = true;
+
+      settings = {
+        highlight = {
+          enable = false;
+          custom_captures = { };
+          additional_vim_regex_highlighting = false;
+        };
+        incremental_selection = {
+          enable = false;
+          keymaps = {
+            init_selection = "gnn";
+            node_incremental = "grn";
+            scope_incremental = "grc";
+            node_decremental = "grm";
+          };
+        };
+        indent = false;
+      };
+    };
```

Comment:
Is this an exhaustive list of defaults?

Reply:
This is just what I saw used in their https://github.com/nvim-treesitter/nvim-treesitter/blob/a8c082b4f5b3b70bde2ffb08cc112dfd7e515fb3/lua/nvim-treesitter/configs.lua#L47-L76 lua config

Reply:
Thanks for the link!

Looks like `highlight.custom_captures` is deprecated now, and they recommend it is set imperatively with a function.

Maybe we should drop the option now then?

Reply:
Also, their `indent` default apperears to be:

```lua
indent = {
  module_path = "nvim-treesitter.indent",
  enable = false,
  is_supported = queries.has_indents,
}
```

Do they document it as `false` elsewhere?

Reply:
The readme mentions they default every module to false and force you to enable the ones you want to use.

Reply:
Looking at some of their other docs, I think some optional options are missing from the table you linked.

E.g. `highlight.disable` (func(lang, buf) -> bool) or (table; listOf lang)

Reply:
Yeah, looks like they have some sync_install and auto_install stuff in the man

Reply:
Going through and cleaning these defaults up, sorry.. taking a little bit of time lol

Reply:
No rush!

Reply:
Alright, think I got it rearranged and updated. Looked like I had a property at top-level instead of under highlight, too.